src/definitions/behaviors/form.js

```diff
@@ -818,26 +818,36 @@ $.fn.form = function(parameters) {
             module.add.field(name, rules);
           },
           field: function(name, rules) {
+            // Validation should have at least a standard format
+            if(validation[name] === undefined || validation[name].rules === undefined) {
+              validation[name] = {
+                rules: []
+              }
```

Comment:
Missing semicolon (although valid and working, but we have it everywhere else...)